resources/compatibility.md

```diff
@@ -5,8 +5,110 @@ shortname: Resources
 title: Browser Compatibility
 ---
 
+<style>
+#browser-compatibility paper-checkbox {
+  pointer-events: none;
+}
+#browser-compatibility th {
+  white-space: nowrap;
+  padding-right: 20px;
+}
+#browser-compatibility .feature-title {
+   white-space: nowrap;
+}
+</style>
+
 Unlike other frameworks, **{{site.project_title}} and its polyfills are intended to work in the _latest
 version_ of "evergreen" browsers**. This means that your favorite browser might not be supported. For more about what that means in practice, check out the [FAQ](faq.html#browsersupport). The support matrix for each polyfill and their
 native browser implementation is below.
 
-<iframe src="https://docs.google.com/spreadsheet/pub?key=0Anye-JMjUkZZdDdoblh6dTlwcWRLQkhKbTVzdHJtcXc&single=true&gid=2&output=html&range=A1:Q43" seamless style="border:none;width:100%;height:830px;"></iframe>
+## Native browser support
+
+<table>
+<thead>
+  <tr><th></th><th>Chrome</th><th>Firefox</th><th>IE 10+</th><th>Safari 8+</th><th>Chrome (Android)</th><th>Safari (iOS 8.1)</th></tr>
+</thead>
+<tr>
```

Comment:
IIRC, this is correct. The browser will do this for you anyway. Can't speak for how consistent this is x-browser however.